frontend/src/components/accountPage/UserListLogic/UserListLogic.tsx

```diff
@@ -0,0 +1,192 @@
+import { FC, useState, useEffect, useRef } from 'react';
+import { Spin } from 'antd';
+import {
+  useTenantsQuery,
+  useApplyTenantMutation,
+} from '../../../generated-types';
+import { getTenantPatchJson } from '../../../graphql-components/utils';
+import UserList from '../UserList/UserList';
+import { User } from '../../../utils';
+
+export interface IUserListLogicProps {
+  workspaceName: string;
```

Comment:
Is this like `workspace-sid`? In that case, that's not exactly the workspace name, but the namespace associated to the workspace, could be better to call it workspaceNamespace then